manifests/cinder-csi-plugin/cinder-csi-nodeplugin.yaml

```diff
@@ -42,6 +42,17 @@ spec:
               mountPath: /csi
             - name: registration-dir
               mountPath: /registration
+        - name: csi-snapshotter
```

Comment:
It should be part of controller-plugin right? Does it needs to be installed on every node?

Reply:
yes, I only do the test ,it works but didn't realize it's in the wrong place, need more analysis on this
thanks for point out